src/lightning/pytorch/core/module.py

```diff
@@ -814,9 +814,10 @@ def validation_step(self, batch, batch_idx, dataloader_idx=0):
             If you don't need to validate you don't need to implement this method.
 
         Note:
-            When the :meth:`validation_step` is called, the model has been put in eval mode
-            and PyTorch gradients have been disabled. At the end of validation,
-            the model goes back to training mode and gradients are enabled.
+            When the :meth:`validation_step` is called, the model has been put
```

Comment:
for completeness can we add a inverse note to the training step, e.g. that when `train_step` is called, the model has NOT been put into train mode and that it is something the user should be aware of